### PR TITLE
Convert api e2e tests to chai

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

# Code review items

- Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- Documented: Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
Originally, was part of another PR, but decided to split for easier review.

**Comment by dianabarsan** [comment]
Please have a look, @SCdF . Thanks!

File: tests/e2e/api/controllers/bulk-get.spec.js [hunk 1]
```diff
@@ -208,9 +218,16 @@ describe('bulk-get handler', () => {
         return utils.requestOnTestDb(offlineRequestOptions);
       })
       .then(result => {
-        expect(result.results.length).toEqual(2);
-        expect(result.results[0].id).toEqual('allowed_task');
-        expect(result.results[1].id).toEqual('allowed_target');
+        chai.expect(result.results).excludingEvery('_rev').to.deep.equal([
```
Comment (SCdF) [inline tests/e2e/api/controllers/bulk-get.spec.js:221]: I learnt something!

Comment (dianabarsan) [reply tests/e2e/api/controllers/bulk-get.spec.js:221]: The absolutely beautiful part is that the properties are excluded from both LHS and RHS compared items.

**Pull request merged**